docs/getting-started.md

```diff
@@ -105,6 +105,33 @@ The `containerd.io` package contains runc too, but does not contain CNI plugins.
 
 To install containerd and its dependencies from the source, see [`BUILDING.md`](/BUILDING.md).
 
+## Installing containerd on Windows
+
+From a PowerShell session run the following commands:
+
+```PowerShell
+# Download and extract desired containerd Windows binaries
+$Version=1.6.2
+curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
+tar.exe xvf .\containerd-windows-amd64.tar.gz
+
+# Copy and configure
+Copy-Item -Path ".\bin\" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
+cd $Env:ProgramFiles\containerd\
+.\containerd.exe config default | Out-File config.toml -Encoding ascii
+# Review the configuration. Depending on setup you may want to adjust:
+# - the sandbox_image (Kubernetes pause image)
+# - cni bin_dir and conf_dir locations
+Get-Content config.toml
+
+# (Optional - but highly recommended) Exclude containerd from Windows Defender Scans
+Add-MpPreference -ExclusionProcess "$Env:ProgramFiles\containerd\containerd.exe"
```

Comment:
This is no longer recommended now that defender has fixed the image validation right?

Reply:
https://github.com/microsoft/Windows-Containers/issues/92#issuecomment-1083591322

Reply:
I didn't realize this was finally fixed!
Yea, I'll remove this.

Reply:
Updated!